mappings/net/minecraft/network/packet/s2c/play/PlayerPositionUpdateS2CPacket.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_pvzlatim net/minecraft/network/packet/s2c/play/PlayerPositionLookS2CPacket
+CLASS net/minecraft/unmapped/C_pvzlatim net/minecraft/network/packet/s2c/play/PlayerPositionUpdateS2CPacket
```

Comment:
This also updates the rotation, so that should also be specified

Reply:
the obvious `PlayerPositionAndRotationUpdate` is a little on the long side, do you have any suggestions?